### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by Sathvika-7284**
Please check the working of the code.
@CynthiaJCU
@ArthurMY

### CODE REVIEW

File: state_names.py [hunk 1]
```diff
@@ -3,18 +3,23 @@
 State names in a dictionary
 File needs reformatting
 """
+
 # TODO: Reformat this file so the dictionary code follows PEP 8 convention
 CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
                 "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
 print(CODE_TO_NAME)
-for key in CODE_TO_NAME:
```
Comment (ArthurMY) [inline state_names.py:10]: the for loop prints out all the code represents place

File: state_names.py [hunk 2]
```diff
@@ -3,18 +3,23 @@
...
-    value = CODE_TO_NAME[key]
-    print(f"{key} is {value}")
 
+all_sentences = ""
 state_code = input("Enter short state: ").upper()
 for i in CODE_TO_NAME:
```
Comment (ArthurMY) [inline state_names.py:14]: this for loop require user enter at least 7 input.

File: word_occurences.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+"""
+Word Occurrences
+Estimate: 30 minutes
+Actual:   1 hour
+"""
+
```
Comment (ArthurMY) [inline word_occurences.py:6]: nice work

File: state_names.py (hunk 2 above)
Comment (Sathvika-7284) [reply state_names.py:14]: Yes these are committed and thus red blocks are not in the actual code now

File: state_names.py (hunk 1 above)
Comment (Sathvika-7284) [reply state_names.py:10]: yes the red blocks of code in not in the actua code

File: word_occurences.py (hunk 3 above)
Comment (Sathvika-7276) [reply word_occurences.py:6]: Thanks!

**Pull request merged**